docs/user-guide/multi-cloud-auth-storage.md

```diff
@@ -424,6 +510,27 @@ genai-bench benchmark \
   --storage-azure-connection-string "DefaultEndpointsProtocol=..."
 ```
 
+**Benchmark OCI GenAI (Grok models) and store in AWS S3:**
```

Comment:
should we provide a doc about Benchmark OCI GenAI and store in OCI bucket instead?